packages/react-components/src/components/CaptionsSettingsModal.tsx

```diff
@@ -95,10 +95,9 @@ export const _CaptionsSettingsModal = (props: _CaptionsSettingsModalProps): JSX.
     if (isCaptionsFeatureActive) {
       onSetSpokenLanguage(languageCode);
     } else {
-      await onStartCaptions({ spokenLanguage: languageCode });
```

Comment:
What is the difference between await and .then()?

Ideally it should be the same
```
foo: ()=> Promise<boolean>;
await foo();
bar();
```

equals 
```
foo: ()=> Promise<boolean>;
foo().then(() => bar(););

```

Reply:
The only difference here is Promise.resolvev way will ignore the error and not throw it, if I remember it right

Reply:
The original implementation with just await throws the above console error. I haven't tried await and then together, I can try that.